cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -109,19 +103,19 @@
 	private final CassandraStorageSettings settings;
 	private final CassandraSemaphore semaphore;
 	private final CradleObjectsFactory objectsFactory;
-	
+

```

Comment:
This ckass is full of "changes" due to presence of stuff like this, i.e. empty line or tabs removal. Please disable this feature in your IDE, if you don't mind.